modules/storage/storage-account/README.md

```diff
@@ -2,67 +2,59 @@
-| `prefix`                          | `string` | No       | Prefix of Storage Account Resource Name. This param is ignored when name is provided.                                                                                                                                                                                                                                                                                                                                                                       |
-| `name`                            | `string` | No       | Name of Storage Account. Must be unique within Azure.                                                                                                                                                                                                                                                                                                                                                                                                       |
-| `subnetID`                        | `string` | No       | ID of the subnet where the Storage Account will be deployed, if virtual network access is enabled.                                                                                                                                                                                                                                                                                                                                                          |
-| `enableVNET`                      | `bool`   | No       | Toggle to enable or disable virtual network access for the Storage Account.                                                                                                                                                                                                                                                                                                                                                                                 |
-| `isZoneRedundant`                 | `bool`   | No       | Toggle to enable or disable zone redundancy for the Storage Account.                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
These two flags are important to me for us to keep. (ping me on teams for detailed explanation)

Reply:
Please see the `Breaking Changes` section in PR description. Ping me anytime for more discussion.